maven/lib/dependabot/maven/file_updater/declaration_finder.rb

```diff
@@ -10,10 +10,14 @@ module Dependabot
   module Maven
     class FileUpdater
       class DeclarationFinder
-        DECLARATION_REGEX =
-          %r{<parent>.*?</parent>|<dependency>.*?</dependency>|
-             <plugin>.*?(?:<plugin>.*?</plugin>.*)?</plugin>|<extension>.*?</extension>|
-             <path>.*?</path>}mx
+        DECLARATION_REGEX = %r{
```

Comment:
`artifactItem` is the only addition here. I believe that this way is more readable